astropy/time/core.py

```diff
@@ -1180,6 +1180,34 @@ def __setitem__(self, item, value):
         self._time.jd1[item] = value._time.jd1
         self._time.jd2[item] = value._time.jd2
 
+    def isclose(self, other, atol=None):
+        """Returns a boolean or boolean array where two Time objects are
+        element-wise equal within a time tolerance.
+
+        Parameters
+        ----------
+        other : `~astropy.time.Time`
+            Time object for comparison.
+        atol : `~astropy.units.Quantity` or `~astropy.time.TimeDelta`
+            Tolerance for equality with units of time (e.g. ``u.s`` or ``u.day``).
+            Default is one bit in the 128-bit JD time representation, equivalent
+            to about 20 picosecs.
+        """
+        if not isinstance(other, Time):
+            raise TypeError("'other' argument must be a Time instance, got "
+                            f'{other.__class__.__name__} instead')
+
+        if atol is None:
+            atol = np.finfo(float).eps * u.day
+
+        if not isinstance(atol, (u.Quantity, TimeDelta)):
+            raise TypeError("'atol' argument must be a Quantity or TimeDelta instance, got "
+                            f'{atol.__class__.__name__} instead')
+
+        # Note: np.abs(self - other) results in an object array of TimeDelta
```

Comment:
There should be no need to do that - `TimeDelta` should be smart enough! (And I think it is - if not, that's a bug...)

Reply:
See #10653

Reply:
As noted there, plain `abs(...)` should work.